velox/functions/lib/string/StringImpl.h

```diff
@@ -514,4 +515,25 @@ FOLLY_ALWAYS_INLINE void pad(
       padPrefixByteLength);
 }
 
+template <typename TOutString>
+FOLLY_ALWAYS_INLINE void toIEEE754_64(TOutString& output, const double& input) {
+  static constexpr auto outLength = 64;
+  output.resize(64);
+  // Cast the double to an unsigned long long pointer
+  unsigned long long* ull = const_cast<unsigned long long*>(
+      reinterpret_cast<const unsigned long long*>(&input));
+  // Convert the unsigned long long to a bitset
+  std::bitset<64> bits = std::bitset<64>(*ull);
+  // Get the sign bit
+  bool sign = bits[63];
+  // Get the exponent bits
```

Comment:
perhaps, add a link to an explanation of this calculation: https://en.wikipedia.org/wiki/Double-precision_floating-point_format

Reply:
made some code changes and I think the link would be needed for from_ieee754_32/64 calculation